scripts/bert/index.rst

```diff
@@ -23,43 +29,144 @@ BERT for Sentence Classification on GLUE tasks
+    $ curl -L https://tinyurl.com/yaznh3os -o download_glue_data.py
+    $ python3 download_glue_data.py --data_dir glue_data --tasks SST
+    $ GLUE_DIR=glue_data python3 finetune_classifier.py --task_name SST --epochs 4 --batch_size 16 --optimizer bertadam --gpu --lr 2e-5 --log_interval 500
 
-It gets validation accuracy of `88.7% <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/finetuned_mrpc.log>`_, whereas the the original Tensorflow implementation give evaluation results between 84% and 88%.
+[3] RTE
 
 .. code-block:: console
 
-   $ MXNET_GPU_MEM_POOL_TYPE=Round GLUE_DIR=glue_data python3 finetune_classifier.py --task_name RTE --batch_size 32 --optimizer bertadam --epochs 3 --gpu  --lr 2e-5
+    $ curl -L https://tinyurl.com/yaznh3os -o download_glue_data.py
+    $ python3 download_glue_data.py --data_dir glue_data --tasks RTE
+    $ GLUE_DIR=glue_data python3 finetune_classifier.py --task_name RTE --batch_size 32 --optimizer bertadam --epochs 3 --gpu  --lr 2e-5
+
+[4] MNLI
 
-It gets RTE validation accuracy of `70.8% <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/finetuned_rte.log>`_
-, whereas the the original Tensorflow implementation give evaluation results 66.4%.
+.. code-block:: console
+
+    $ curl -L https://tinyurl.com/yaznh3os -o download_glue_data.py
+    $ python3 download_glue_data.py --data_dir glue_data --tasks MNLI
+    $ GLUE_DIR=glue_data python3 finetune_classifier.py --task_name MNLI --max_len 80 --log_interval 100 --epsilon 1e-8 --gpu
 
 Some other tasks can be modeled with `--task_name` parameter.
 
+BERT for Question Answering on SQuAD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-----------------------+---------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+
+|                       |                                                            SQuAD 1.1                                                            |                                                             SQuAD 1.1                                                            |                                                             SQuAD 2.0                                                            |
++-----------------------+---------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Could you please help provide references for the models in this table? Thanks
#1184

Reply:
https://arxiv.org/abs/1810.04805